src/storage/txn/flow_controller.rs

```diff
@@ -772,22 +778,7 @@ impl<E: KvEngine> FlowChecker<E> {
                     .with_label_values(&[cf, "tick_spare"])
                     .inc();
 
-                let throttle = if checker.long_term_num_l0_files.get_avg()
-                    >= self.l0_files_threshold as f64 * 0.5
-                    || checker.long_term_num_l0_files.get_recent() as f64
-                        >= self.l0_files_threshold as f64 * 0.5
-                    || checker.last_num_l0_files_from_flush >= self.l0_files_threshold
```

Comment:
Not needed?

Reply:
Yes, after a second thought, we can simply speed up when no compaction/flush happens in 30s. These conditions just make it complex.